htdocs/api/v0.0.3-dev/candidates/visits/images/headers/Headers.php

```diff
@@ -108,7 +115,13 @@ public function handleGET()
                                          "ZSpace"        => $ZSpace,
                                          "TimeDimension" => $TimeD,
                                         ],
-
+                       'ScannerInfo' => [
+                                         "Manufacturer"    => $Manufacturer,
+                                         "Model"           => $Model,
+                                         "SoftwareVersion" => $SoftwareVersion,
+                                         "SerialNumber"    => $SerialNumber,
+                                         "FieldStrength"   => $FieldStrength,
```

Comment:
change variables name according to previous comment

Reply:
done!